.github/workflows/coverage.yaml

```diff
@@ -66,7 +66,7 @@ jobs:
         shell: bash
         run: |
           source ./dozer-tests/python_udf/virtualenv.sh
-          cargo test --all-features --no-fail-fast
+          cargo test --all-features --no-fail-fast --features python
```

Comment:
does it not work with `all-features` ? do you need to explicitly include python ? Can you check. ?

Reply:
My mistake, I didn't notice `all-features` 🤦